manifests/config/resource_source.pp

```diff
@@ -83,8 +83,12 @@
   $puppet_enterprise_host             = '',
   $puppet_enterprise_port             = '',
   $puppet_enterprise_ssl_dir          = '',
+  $puppet_enterprise_certificate_name = '',
```

Comment:
we should default to undef instead of empty string. this would be the current best practice